qlib/backtest/decision.py

```diff
@@ -576,3 +576,16 @@ def __repr__(self) -> str:
             f"trade_range: {self.trade_range}; "
             f"order_list[{len(self.order_list)}]"
         )
+
+
+class TradeDecisionWithDetails(TradeDecisionWO):
```

Comment:
Add some explanations on why (in what scenarios) we need this.

Reply:
Done.